src/iocore/cache/unit_tests/test_Stripe.cc

```diff
@@ -0,0 +1,159 @@
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#include "main.h"
+
+#include <array>
+#include <ostream>
+
+// Required by main.h
+int cache_vols            = 1;
+bool reuse_existing_cache = false;
+
+struct AddWriterBranchTest {
+  int initial_buffer_size{};
+  int agg_len{};
+  int header_len{};
+  int write_len{};
+  int readers{};
+  bool result{};
+};
+
+std::array<AddWriterBranchTest, 32> add_writer_branch_test_cases = {
+  {
+   {0, 0, 0, 0, 0, true},
+   {0, 0, 0, 0, 1, true},
+   {0, 0, 0, 1, 0, true},
+   {0, 0, 0, 1, 1, true},
+   {0, 0, MAX_FRAG_SIZE + 1 - sizeof(Doc), 0, 0, false},
+   {0, 0, MAX_FRAG_SIZE + 1 - sizeof(Doc), 0, 1, false},
+   {0, 0, MAX_FRAG_SIZE + 1 - sizeof(Doc), 1, 0, false},
+   {0, 0, MAX_FRAG_SIZE + 1 - sizeof(Doc), 1, 1, false},
+   {0, AGG_SIZE + 1, 0, 0, 0, false},
+   {0, AGG_SIZE + 1, 0, 0, 1, false},
+   {0, AGG_SIZE + 1, 0, 1, 0, false},
+   {0, AGG_SIZE + 1, 0, 1, 1, false},
+   {0, AGG_SIZE + 1, MAX_FRAG_SIZE + 1 - sizeof(Doc), 0, 0, false},
+   {0, AGG_SIZE + 1, MAX_FRAG_SIZE + 1 - sizeof(Doc), 0, 1, false},
+   {0, AGG_SIZE + 1, MAX_FRAG_SIZE + 1 - sizeof(Doc), 1, 0, false},
+   {0, AGG_SIZE + 1, MAX_FRAG_SIZE + 1 - sizeof(Doc), 1, 1, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, 0, 0, 0, 0, true},
+   {AGG_SIZE + cache_config_agg_write_backlog, 0, 0, 0, 1, true},
+   {AGG_SIZE + cache_config_agg_write_backlog, 0, 0, 1, 0, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, 0, 0, 1, 1, true},
+   {AGG_SIZE + cache_config_agg_write_backlog, 0, MAX_FRAG_SIZE + 1 - sizeof(Doc), 0, 0, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, 0, MAX_FRAG_SIZE + 1 - sizeof(Doc), 0, 1, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, 0, MAX_FRAG_SIZE + 1 - sizeof(Doc), 1, 0, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, 0, MAX_FRAG_SIZE + 1 - sizeof(Doc), 1, 1, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, AGG_SIZE + 1, 0, 0, 0, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, AGG_SIZE + 1, 0, 0, 1, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, AGG_SIZE + 1, 0, 1, 0, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, AGG_SIZE + 1, 0, 1, 1, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, AGG_SIZE + 1, MAX_FRAG_SIZE + 1 - sizeof(Doc), 0, 0, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, AGG_SIZE + 1, MAX_FRAG_SIZE + 1 - sizeof(Doc), 0, 1, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, AGG_SIZE + 1, MAX_FRAG_SIZE + 1 - sizeof(Doc), 1, 0, false},
+   {AGG_SIZE + cache_config_agg_write_backlog, AGG_SIZE + 1, MAX_FRAG_SIZE + 1 - sizeof(Doc), 1, 1, false},
+   }
```

Comment:
I'm wondering why this array is not formatted by clang-format 🤔

Reply:
How can you tell that it isn't? Perhaps the file is accidentally whitelisted somehow? I will look into this before merging.

Reply:
I ran the clang-format locally with the `--target clang-format` option to `cmake`, and I confirmed that local changes to `test_Stripe.cc` do get formatted correctly. I think this is enough evidence that the formatter is running on the file; but maybe it is ignoring that array for some reason. I will merge for now, but I will make a note of this.